File: Python codes/notasfor.py
media = 0.0
somaNotas = 0.0


for i in range(4):
    nota = int(input("insira um número"+(i+1)+":"))
    somaNotas += nota
    if nota <0 & nota >10:
        print("inválidos")

media = notas /4
print("sua média é: ", media)

File: Aulas PA/js/notasfor.js
var somaNotas;
var notas;
let media;
for(let i = 0; i <4; i++){
    notas = parseFloat(prompt("insira a nota"+ (i+1)+":"));
    somaNotas += notas

    if(notas >10 || nota < 0 || !notas.isNaN()){
        console.log("inválidas")
        i -=1
    }

    }
    media = notas / 4;
    console.log("sua média é: "+ media);
    